interface/patient_file/encounter/delete_form.php

```diff
@@ -77,7 +77,7 @@
 ?>
 <input type="hidden" id="confirm" name="confirm" value="1"/>
 <p>
-<?php echo xlt('You are about to delete the following form from this encounter') . ': ' . text($_GET['formname']); ?>
+<?php echo xlt('You are about to delete the following form from this encounter') . ': ' . xlt(text($_GET['formname'])); ?>
```

Comment:
Thank you very much for the reading material !
I will also transfer this to all the developers in my team  :)